bin/oref0-pump-loop.sh

```diff
@@ -690,7 +690,11 @@ function wait_for_bg {
 
 function glucose-fresh {
     # set mtime of monitor/glucose.json to the time of its most recent glucose value
-    touch -d "$(date -R -d @$(jq .[0].date/1000 monitor/glucose.json))" monitor/glucose.json 2>&3
+    if grep "MDT cgm" openaps.ini 2>&3 >&4; then
```

Comment:
Would it be better to check for the existence of `.[0].display_time` here?  FWIW, my data from Dexcom includes both fields.

Reply:
This seems to be the check used to work out if it's MDT data or not elsewhere in the script, I was trying to be consistent. Do you have any preference?

Reply:
My Dexcom data has a display_time field as well, so it's not really a MDT-specific thing.  I think we should be able to just use display_time across the board, but probably safer for backwards compatibility to fall back to date if display_time doesn't exist?

Reply:
Ok, I'll try to take a look at it tomorrow.